client-react/src/signin/index.js

```diff
@@ -133,7 +151,8 @@ const SignIn = () => {
     <CenterContainer>
       <ContentWrapper display={page[0] ? true : false}>
         <LeftContainer img={Polygon} light>
-          <Title>Sign In</Title>
+          <Branding src={logo} alt="logo" />
+          <Title style={{ marginTop: "40px" }}>Sign In</Title>
```

Comment:
rather than doing inline styling. lets pass in a prop and let styled-components handle which style needs to be applied